datalad_container/containers_run.py

```diff
@@ -131,14 +132,34 @@ def __call__(cmd, container_name=None, dataset=None,
             # just prepend and pray
             cmd = container['path'] + ' ' + cmd
 
+        try:
+            extra_inputs = json.loads(container.get("extra-inputs","[]") or "[]")
+        except json.JSONDecodeError as e:
+            lgr.error((
+                "extra-inputs of container ({!r}) is not JSON. "
+                "Error: {}. Continuing without it."
+            ).format(container.get("extra-inputs"),e))
+            extra_inputs = []
+
+        if not isinstance(extra_inputs, (str,list,tuple)):
+            lgr.warning((
+                "extra-inputs of container ({!r}) is not a JSON array or string "
+                "but a {} ({!r}). "
+                "This might result in unexpected behaviour."
+            ).format(container.get("extra-inputs"),type(extra_inputs),extra_inputs))
+        extra_inputs = list(map(str, 
+            [extra_inputs] if isinstance(extra_inputs, str) else extra_inputs
+        ))
```

Comment:
if we just store multiple values in the config in multiple lines, then smth like `list(config.get("extra-inputs", [], get_all=True))` should be all what is needed since by default values are `str` already and you might just get a tuple not a list may be